docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/gitea/index.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_label: Gitea
+---
+
+# The Gitea Webhook Receiver
+
+The Gitea Webhook Receiver will respond to `push` events originating from
+Gitea repositories.
+
+In response to a `push` event, the receiver "refreshes" Warehouses subscribed to
+the Git repository from which the event originated.
+
+:::info
+"Refreshing" a `Warehouse` resource means enqueuing it for immediate
+reconciliation by the Kargo controller, which will execute the discovery of
+new artifacts from all repositories to which that `Warehouse` subscribes.
+:::
+
+## Configuring the Receiver
+
+The Gitea webhook receiver will need to reference a Kubernetes `Secret` with a
+`secret` key in its data map. This
+[shared secret](https://en.wikipedia.org/wiki/Shared_secret) will be used by
+Gitea to sign requests. The receiver will use it to authenticate those requests
+by verifying their signatures.
+
+:::note
+The following commands are suggested for generating and base64-encoding a
+complex secret:
+
+```shell
+secret=$(openssl rand -base64 48 | tr -d '=+/' | head -c 32)
+echo "Secret: $secret"
+echo "Encoded secret: $(echo -n $secret | base64)"
+```
+:::
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Project
+metadata:
+  name: kargo-demo
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gitea-wh-secret
+  namespace: kargo-demo
+data:
+  secret: <base64-encoded secret>
+---
+apiVersion: kargo.akuity.io/v1alpha1
+kind: ProjectConfig
+metadata:
+  name: kargo-demo
+  namespace: kargo-demo
+spec:
+  webhookReceivers: 
+    - name: gitea-wh-receiver
+      gitlab:
+        secretRef:
+          name: gitea-wh-secret
+```
+
+## Retrieving the Receiver's URL
+
+Kargo will generate a hard-to-guess URL from the configuration. We can obtain
+this URL using the following command:
+
+```shell
+kubectl get projectconfigs kargo-demo \
+  -n kargo-demo \
+  -o=jsonpath='{.status.webhookReceivers}'
+```
+
+## Registering with Gitea
+
+1. Navigate to `https://gitea.com/<namespace>/<repo>/settings/hooks`, where
+   `<namespace>` has been replaced with a Gitea username or group name and
+   `<project>` has been replaced with the name of a project belonging to that
+   namespace and for which you are an administrator.
```

Comment:
I'm actually impressed with Gitea's array of options for registering webhooks. They can be registered at the repo level, org level, or even user level. Fortunately, the form seems to be the same for all cases.

As far as I can tell, all of these work:

* Repo level: `https://gitea.com/<namespace>/<repo>/settings/hooks` (exactly as you describe above)
* All repos belonging to an org: `https://gitea.com/org/<org>/settings/hooks`
* All repos belonging to a user: `https://gitea.com/user/settings/hooks`